lte/protos/oai/nas_state.proto

```diff
@@ -376,6 +376,14 @@ message EsmEbrContext {
   Timer timer = 100; // nas_timer_t
 }
 
+// t3422_arg is nw_detach_data_t
+ message NwDetachData {
+  uint32 ue_id = 1;
+  bool timer_running = 2;
```

Comment:
No need for this flag, it can be checked with if NwDetachData field is non-null in EmmContext message.